pkg/controller/controller.go

```diff
@@ -485,11 +485,13 @@ func (p *csiProvisioner) Provision(ctx context.Context, options controller.Provi
 		case pvcKind:
 			rc.clone = true
 		default:
-			klog.Infof("DataSource specified (%s) is not supported by the provisioner, waiting for an external data populator to create the volume", options.PVC.Spec.DataSource.Kind)
 			// DataSource is not VolumeSnapshot and PVC
-			// Wait for an external data populator to create the volume
-			p.eventRecorder.Event(options.PVC, v1.EventTypeNormal, "Provisioning", fmt.Sprintf("Waiting for a volume to be created by an external data populator"))
-			return nil, controller.ProvisioningFinished, nil
+			// Assume external data populator to create the volume, and there is no more work for us to do
+			p.eventRecorder.Event(options.PVC, v1.EventTypeNormal, "Provisioning", fmt.Sprintf("Assuming an external populator will provision the volume"))
```

Comment:
We agreed to reword this event text at the weekly data populators meeting today. The exact wording is still up for debate, but I wanted to choose language that made no claims about whether an external populator exists or not, only that the sidecar is passing on this PVC. An additional event will be created by the populator itself, if one exists, or by the central populator-controller if one does not.

Reply:
Looks good now.